src/main/java/bio/terra/cli/command/app/passthrough/Git.java

```diff
@@ -68,8 +94,30 @@ private void validateCloneCommand() {
     }
   }
 
+  private @Nullable Workspace attemptToGetDataCollectionWorkspace(DataCollection dataCollection) {
+    try {
+      return dataCollection.getDataCollectionWorkspace();
+    } catch (SystemException e) {
+      // If a user does not have access to the data collection, do not throw.
+      logger.warn(String.format("Failed to get Data collection %s", dataCollection.getName()));
+      return null;
+    }
+  }
+
+  private void getGitRepoResourceToClone(List<Resource> resources) {
```

Comment:
rename this? this method isn't getting git repos, it's executing the clone commands